src/unit_tests/syscheckd/test_run_check.c

```diff
@@ -219,6 +225,42 @@ static int teardown_max_fps(void **state) {
     return 0;
 }
 
+#ifdef TEST_WINAGENT
+
+typedef struct _win32rtfim {
+    HANDLE h;
+    OVERLAPPED overlap;
+
+    char *dir;
+    TCHAR buffer[65536];
+    unsigned int watch_status;
+} win32rtfim;
+
+extern void free_win32rtfim_data(win32rtfim *data);
```

Comment:
Can we move this declaration to the top of the file where there are other declarations?